### DISCUSSION TITLE
failed to wait for sqlite3 dump command: exit status 1

### DISCUSSION THREAD

**Discussion by ColonelBlimp** [aarondl/sqlboiler#1098] [Q&A] [answered]
Hi, I have been following the instructions (README.md), and I have got as far as the models being created. However, when I run

`go test ./models`

I receive the following error:

`Unable to execute setup: failed to wait for sqlite3 dump command: exit status 1`

I have created a `tables_schema.sql` with the `CREATE TABLE' sql.  I have tried locating it in `./models` and in the root of my app - no joy.  Can anyone give me some pointers as to what the issue is?

------------------
Okay, so tables_schema.sql has nothing to do with the issue.

I have checked the sqlite3_main_test.go the following:

dumpCmd := exec.Command("sqlite3", "-cmd", ".dump", s.dbName)

I have tested on the command line (use Windows): sqlite3 -cmd .dump ./db/data.db and it works just fine?

I am new to Go and sqlboiler. Any help appreciated :-)

------------------
Further update: I moved the database to the root of my app and things progressed further. Now I have the following similar issue:

`Unable to execute setup: failed to wait for sqlite3 create command: exit status 1`

Again, tried the `tables_create.sql` in the root and '/models` folder... - no joy.

**Comment by stephenafamo** [comment, marked as answer]
The auto-generated tests have been a little bit neglected for a while now. It is a bit difficult to generate given how varied the database structures could be and right now. We may consider removing it altogether in the near future.

I am not quite sure what is causing your specific issue, but I am hesitant to debug this myself.

While not a solution, perhaps you can just generate your models with the `--no-tests` flag, and generally test the rest of your code that the models are working as expected.

**Reply by ColonelBlimp** [reply]
Thank you for the reply. I tried to debug, but it was too much... And I don't have the bandwidth now. I will try as you suggest. Many thanks again...😊
